SECURITY.md

```diff
@@ -0,0 +1,35 @@
+# Security
+
+Target takes the security of our software products and services seriously, which includes all source code repositories managed through [our GitHub organizations](https://opensource.target.com).
+
+If you believe you have found a security vulnerability in any Target-owned repository that meets Target's definition of a security vulnerability, please report it to us as described below.
+
+## Reporting Security Issues
+
+**Please do not report security vulnerabilities through public GitHub issues.**
+
+Instead, please report them to the Target Cyber Security team at _____.
+
+If you prefer to submit without logging in, send an email to _____. If possible, encrypt your message with our PGP key; please download it from _____.
```

Comment:
do we really have a PGP key process?

Reply:
@bmuenzenmeyer - as far as I know we don't have a PGP process in place. I think there might be a way to assign something at the org level - but I'm not familiar with it.

Reply:
Removed for now